### PR TITLE
validate phone number

### PR DESCRIPTION
**Pull request by baali-boudjemaa**

### CODE REVIEW

File: android/app/build.gradle
Comment (OsamaBentaib) [inline android/app/build.gradle:1]: Why is this change necessary?

File: pubspec.yaml
Comment (OsamaBentaib) [inline pubspec.yaml:1]: Why did you add the space here?

File: devtools_options.yaml
Comment (OsamaBentaib) [inline devtools_options.yaml:1]: What is this for?

**Review by OsamaBentaib — CHANGES_REQUESTED** [review]
Please check the comments

File: pubspec.yaml
Comment (baali-boudjemaa) [reply pubspec.yaml:1]: i  added it accidentally

File: devtools_options.yaml
Comment (OsamaBentaib) [reply devtools_options.yaml:1]: You didn't resolve this why??

**Comment by OsamaBentaib** [comment]
Please fix all the comments i asked for because I see no changes

**Comment by baaliboudjemaa** [comment]
> Please fix all the comments i asked for because I see no changes

fixed

File: android/app/build.gradle
Comment (baali-boudjemaa) [reply android/app/build.gradle:1]: There was a conflict in libc++_shared.so module

File: devtools_options.yaml
Comment (baali-boudjemaa) [reply devtools_options.yaml:1]: i delete it

**Pull request merged**